draft-ietf-quic-recovery.md

```diff
@@ -634,32 +650,48 @@ are detected lost.
 ~~~
    OnPacketsLost(lost_packets):
      largest_lost_packet = lost_packets.last()
+     // Start a new recovery epoch if the lost packet is larger
+     // than the end of the previous recovery epoch.
      if (end_of_recovery < largest_lost_packet.packet_number):
        end_of_recovery = largest_sent_packet
        congestion_window *= kLossReductionFactor
+       congestion_window = max(congestion_window, kMinimumWindow)
```

Comment:
nit: combine this line with the previous one:
congestion_window = 
    max(congestion_window * kLossReductionFactor, kMinimumWindow)